### PR TITLE
Add Rubocop

### PR DESCRIPTION
**Pull request by the-codetrane**
Adding linting to the app to unify coding style and sanitize code base.

### CODE REVIEW

**Comment by the-codetrane** [comment]
@mnyrop I'm _reasonably_ sure these changes are merely syntactical, but I do think we should QA it to make sure I didn't, in fact, introduce any bugs. Do we do that? If so, how?

**Comment by mnyrop** [comment]
@the-codetrane ci tests pass, rubocop (as current configured in the rubocop.yml) [fails](https://github.com/NYULibraries/spatial_data_repository/actions/runs/5337843303/jobs/9674409792?pr=264#step:4:2).

I'll leave it up to you if you'd like to make rubocop pass before merging this PR? Or if you'd rather temporarily re-disable its invocation in the workflow yaml while you power through other tickets–that's fine too. Either way, approving this PR now!

**Review by mnyrop — APPROVED** [review]

**Pull request merged**
